spec/stubs/dummy_stub.yml

```diff
@@ -6,94 +6,68 @@ producer: core
   POST:
     With valid card details:
       request:
-        body: |
-          {
-            "card_number": "4111111111111111",
-            "exp_month": "08",
-            "exp_year": "2017"
-          }
+        card_number: "4111111111111111"
+        exp_month: "08"
+        exp_year: "2017"
       response:
         code: 200
-        body: |
-          {
-            "token": "CTABCDEFG",
-            "exp_month": "08",
-            "exp_year": "2017",
-            "status": "ready"
-          }
+        body:
+          token: "CTABCDEFG"
+          exp_month: "08"
+          exp_year: "2017"
+          status: "ready"
     With expired card:
       request:
-        body: |
-          {
-            "card_number": "411111111",
-            "exp_month": "01",
-            "exp_year": "2000"
-          }
+        card_number: "411111111"
+        exp_month: "01"
+        exp_year: "2000"
       response:
         code: 400
-        body: |
-          {
-            "error": "validation_error",
-            "message": "card expired"
-          }
+        body:
+          error: "validation_error"
+          message: "card expired"
 /v1/cards/CTabcdef:
   GET:
     With card existing:
       response:
         code: 200
-        body: |
-          {
-            "token": "CTabcdef",
-            "status": "ready"
-          }
+        body:
+          token: "CTabcdef"
+          status: "ready"
     With non-existent card:
       response:
         code: 404
-        body: |
-          {
-            "error": "validation_error",
-            "message": "card not found"
-          }
+        body:
+          error: "validation_error"
+          message: "card not found"
   DELETE:
     With card existing:
       response:
         code: 200
-        body: |
-          {
-            "token": "CTabcdef",
-            "status": "deleted"
-          }
+        body:
+          token: "CTabcdef"
+          status: "deleted"
 /v1/cards/CTabcdef/credits:
   POST:
     With card existing:
       request:
-        body: |
-          {
-            "amount_cents": 100
-          }
+        amount_cents: 100
       response:
         code: 200
-        body: |
-          {
-            "token": "CCaoeu",
-            "amount_cents": 100
-          }
+        body:
+          token: "CCaoeu"
+          amount_cents: 100
 /v1/cards/CTabcdef/customers:
   PUT:
     Valid customer:
       request:
-        body: |
-          {
-            "name": "John Smith",
-            "city": "San Francisco",
-            "state": "CA"
-          }
+        name: "John Smith"
```

Comment:
This wouldn't be a valid Rester request `PUT /v1/cards/CTabcdef/customers`.  You can only `PUT` to a resource instance (with an identifier) not the resource itself.  For example: `PUT /v1/cards/CTabcdef/customers/CUabcdef`